index.bs

```diff
@@ -7,3 +7,514 @@ URL: https://netwerk-digitaal-erfgoed.github.io/requirements-network-of-terms/
+        </tr>
+        <tr>
+            <td>Name</td>
+            <td>The name of the terminology source, e.g. "Dutch World War Two Thesaurus"<td>
+        </tr>
+        <tr>
+            <td>Alternate name</td>
+            <td>The altername name of the terminology source, if any. E.g. "WW2 Thesaurus"<td>
+        </tr>
+        <tr>
+            <td>Name of organization</td>
+            <td>The name of the organization that manages the terminology source, e.g. "NIOD Institute for War, Holocaust and Genocide Studies"<td>
+        </tr>
+        <tr>
+            <td>Alternate name of organization</td>
+            <td>The altername name of the organization that manages the terminology source, if any. E.g. "NIOD"<td>
+        </tr>
+        <tr>
+            <td>Description</td>
+            <td>A description of the terminology source, e.g. "The WW2 Thesaurus is a validated, hierarchically organized keyword list for the thematic disclosure of (digital) sources from the period 1940-1945"<td>
+        </tr>
+    </tbody>
+</table>
+
+This information is published in all languages that the terminology source supports.
+This enables multilingual applications to use the information.
+
+The Network of Terms publishes this information via an endpoint. This allows applications to process the information automatically.
+As with [requirement 1](#1) the endpoint will use [GraphQL](https://graphql.org/) as its interface and \[JSON](https://en.wikipedia.org/wiki/JSON) as its data format.
+For publishing information about the managing organizations [Schema.org](https://schema.org/) is used as metadata schema.
+
+<div class="note">
+    **Discussion**: is it useful to also publish this information as RDF with e.g. DCAT or Schema.org as metadata schemas?
+    In keeping with the [[!NDE-DATASETS|Requirements for Datasets]], this allows applications to use the information with Linked Data technology.
+</div>
+
+The Network of Terms can search in one and more terminology sources {#5}
+--------------------
+**Priority**: Must have
+
+Some users want the Network of Terms to find terms in a *particular* terminology source.
+For example: a user may want to use the WW2 Thesaurus because it best describes the user's heritage information.
+The Network of Terms should support this by having users indicate which source must be queried.
+
+At the same time, other users want the Network of Terms to find terms in *multiple* terminology sources.
+Users want to perform these searches as efficiently as possible: they do not want to have to submit their search query multiple times to multiple sources, but once.
+The Network of Terms should support this by having users indicate which sources must be queried.
+The Network of Terms then searches these sources and returns all matching terms in one result set.
+
+For optimal performance the Network of Terms searches terminology sources parallelly, not sequentially.
```

Comment:
‘in parallel’ or ‘concurrently’ (though the latter has a slightly different meaning) sounds a bit more natural.

Reply:
👍🏻